style.css

```diff
@@ -273,91 +273,64 @@ header {
 	margin-left: 10px;
 	font-size: 1.4rem;
 	border-bottom: 4px solid rgba(158, 56, 28, 0.8);
+	display: inline-block;
+	line-height: 1.5;
 }
 
 .projectLink:hover {
 	border-bottom: 4px solid #FFFFFF;
 }
 
-
-/* Possible extra section --
-.featureProjects {
-	width: 100vw;
-	height: 400px;
-	display: flex;
-}
-
-.featureProjectsItem {
-	background-image: url("#");
-	background-size: cover;
-}
-
--------- */
-
 /* -- Contact -- */
 
 .contactMe {
 	display: flex;
-	justify-content: space-between;
-	align-items: flex-end;
-	background-color: DarkGray;
-	height: 200px;
+	flex-direction: column;
+	gap: 20px;
+	align-items: baseline;
+	background-color: #808080;
 	padding: 40px 10%;
-}
-
-.mailIcon {
-	font-size: 9rem;
-}
-
-.contactText {
-	font-size: 1.9rem;
+	box-shadow: inset 0px 12px 24px -12px rgba(0, 0, 0, 0.5);
+	-webkit-box-shadow: inset 0px 12px 24px -12px rgba(0, 0, 0, 0.5);
```

Comment:
should box-shadow go below -webkit-box-shadow?

Reply:
Yeah, good spot. I'll fix this.